<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="target_energy_lineedit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Enter desired energy in eV.</string>
          </property>
          <property name="placeholderText">
           <string>Target energy (eV)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>eV</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="edge_combo_box">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Select an edge, or type one in the form &quot;Na L3&quot;, etc.</string>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>Ni K (8333 eV)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="set_energy_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Queue a plan to change the X-ray energy.</string>
          </property>
          <property name="text">
           <string>Set Energy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Energy:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Edge:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Monochromator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="PyDMByteIndicator" name="PyDMByteIndicator_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="offColor" stdset="0">
             <color>
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </property>
            <property name="showLabels" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PyDMPushButton" name="PyDMPushButton_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Disable</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="PyDMByteIndicator" name="PyDMByteIndicator">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="offColor" stdset="0">
             <color>
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </property>
            <property name="showLabels" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PyDMPushButton" name="PyDMPushButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>Disable</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="PyDMEnumComboBox" name="PyDMEnumComboBox">
          <property name="toolTip">
           <string/>
          </property>
          <property name="channel" stdset="0">
           <string>${MONO_MODE_PV}</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Energy:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="PyDMLabel" name="PyDMLabel_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>8000 eV</string>
          </property>
          <property name="precision" stdset="0">
           <number>2</number>
          </property>
          <property name="showUnits" stdset="0">
           <bool>true</bool>
          </property>
          <property name="precisionFromPV" stdset="0">
           <bool>false</bool>
          </property>
          <property name="channel" stdset="0">
           <string>${MONO_ENERGY_PV}</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Feedback χ:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Mode:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Feedback θ:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="mono_caqtdm_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Launch caQtDM panel for this mono.</string>
          </property>
          <property name="text">
           <string>Advanced</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Insertion Device</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="1" column="1">
         <widget class="PyDMLabel" name="PyDMLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Current set point for the ID (in keV).</string>
          </property>
          <property name="text">
           <string>8.14 keV</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="precision" stdset="0">
           <number>0</number>
          </property>
          <property name="showUnits" stdset="0">
           <bool>true</bool>
          </property>
          <property name="precisionFromPV" stdset="0">
           <bool>true</bool>
          </property>
          <property name="channel" stdset="0">
           <string>${ID_PREFIX}:Energy.VAL</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Energy:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Readback:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="id_caqtdm_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Launch caQtDM panel for this ID.</string>
          </property>
          <property name="text">
           <string>Advanced</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Taper:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="PyDMLabel" name="PyDMLabel_2">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(75, 75, 75)</string>
          </property>
          <property name="text">
           <string>22.3 mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="showUnits" stdset="0">
           <bool>true</bool>
          </property>
          <property name="channel" stdset="0">
           <string>${ID_PREFIX}:Gap.VAL</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(75, 75, 75)</string>
          </property>
          <property name="text">
           <string>Gap:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="PyDMLabel" name="PyDMLabel_4">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>0.1 keV</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="channel" stdset="0">
           <string>${ID_PREFIX}:TaperEnergy.VAL</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="PyDMLabel" name="PyDMLabel_5">
          <property name="toolTip">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(75, 75, 75)</string>
          </property>
          <property name="text">
           <string>1.3 mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="channel" stdset="0">
           <string>${ID_PREFIX}:TaperGap.VAL</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMByteIndicator</class>
   <extends>QWidget</extends>
   <header>pydm.widgets.byte</header>
  </customwidget>
  <customwidget>
   <class>PyDMEnumComboBox</class>
   <extends>QComboBox</extends>
   <header>pydm.widgets.enum_combo_box</header>
  </customwidget>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
